docs/src/tutorial.md

```diff
@@ -201,35 +194,41 @@ ax.title = "input timeseries"
 fig
 ```
 
-Then we decide what indicators and change metrices to use
+Then we decide what indicators and change metrics to use and run the analysis.
 
 ```@example MAIN
-# these indicators are suitable for Critical Slowing Down
+# These indicators are suitable for Critical Slowing Down
 indicators = [var, ar1_whitenoise]
-ind_conf = IndicatorsConfig(indicators; width = 400)
+indconfig = IndicatorsConfig(tfluct, indicators; width = 400)
+
 
-# use spearman correlation for both indicators
-change_metric = spearman
-sig_conf = SignificanceConfig(change_metric; width = 20, n_surrogates = 1000)
+# use the ridge regression slope for both indicators
+change_metrics = [RidgeRegressionSlope()]
+sigconfig = SignificanceConfig(indconfig, change_metrics; width = 30, n_surrogates = 1000)
 
 # perform the full analysis
-result = indicators_analysis(x_nl_fluct, ind_conf, sig_conf)
+results = indicators_analysis(tfluct, x_nl_fluct, indconfig, sigconfig)
 ```
 
-And lastly, obtain some flags for when there is a significant indicator change
+That's it, just a couple of lines! To get insight on the results, we plot the obtained p-values vs. the original time series:
+
 ```@example MAIN
-sig = indicators_significance(result, 0.99)
-
-# Plot the original timeseries
-fig, ax = lines(tfluct, x_nl_fluct; label = "input")
-# Scatter the significance of each indicator
-for i in size(sig, 2)
-    signif_idxs = findall(sig[:, i])
-    isempty(signif_idxs) && continue
-    # get timepoints in real time
-    tflags = tfluct[result.t_change[signif_idxs]]
-    vlines!(ax, tflags; label = "indicator $(indicators[i])", color = Cycled(1+i))
-end
-axislegend()
+fig, ax = lines(tfluct, x_nl_fluct; color = Cycled(2), label = "input")
+axpval, = lines(fig[2,1], sigconfig.t_change, results.pval[:, 1]; color = Cycled(3), label = "p-value of var")
+lines!(axpval, sigconfig.t_change, results.pval[:, 2]; color = Cycled(4), label = "p-value of ar1")
+xlims!(ax, (0, 50))
+xlims!(axpval, (0, 50))
+axislegend(axpval)
 fig
 ```
+
+Thresholding the p-value results in a loss of information and it is therefore preferable to be avoided. For automation purposes it might however be unavoidable. Here we show an example where we consider that both the variance and the AR1-regression coefficient need to show a p-value below 0.05 for a transition to be indicated in a binary sense:
```

Comment:
Hm, I think we should provide some automation here. We should have a function `transition_flags` that returns the flags (timesstamps) of significance of each indicator, as well as one additional that is the boolean and of all flags.